app/routes/collections.photographs.$photograph.tsx

```diff
@@ -5,6 +5,8 @@ import IIIFViewer from "~/components/layout/IIIFViewer.client";
 import { ClientOnly } from "remix-utils/client-only";
 import type { LoaderFunctionArgs } from "@remix-run/node";
 import type { ESPhotographItem } from "~/esTypes";
+import { FontAwesomeIcon } from "@fortawesome/react-fontawesome";
```

Comment:
This super nit-picky, but I prefer to have all the type imports grouped at the end. It's also a best practice to  have all the imports from external libraries first, then internal imports, and finally type imports. As you can see, I haven't followed that here, but it's something I clean up whenever I notice.